File: vision/python version/import_dataset.py
from google.cloud import automl

# TODO(developer): Uncomment and set the following variables
project_id = "image-prediction-326402"
dataset_id = "ICN2734200095011504128"
path = "gs://image-prediction-326402-vcm/csv/all_data.csv"

client = automl.AutoMlClient()
# Get the full path of the dataset.
dataset_full_id = client.dataset_path(project_id, "us-central1", dataset_id)
# Get the multiple Google Cloud Storage URIs
input_uris = path.split(",")
gcs_source = automl.GcsSource(input_uris=input_uris)
input_config = automl.InputConfig(gcs_source=gcs_source)
# Import data from the input URI
response = client.import_data(name=dataset_full_id, input_config=input_config)

print("Processing import...")
print("Data imported. {}".format(response.result()))

File: vision/import_dataset.js
/**
 * TODO(developer): Uncomment these variables before running the sample.
 */
const projectId = 'image-prediction-326402';
const location = 'us-central1';
const datasetId = 'ICN9196302610334744576';
const path = 'gs://image-prediction-326402-vcm/csv/all_data.csv';

// Imports the Google Cloud AutoML library
const {AutoMlClient} = require('@google-cloud/automl').v1;

// Instantiates a client
const client = new AutoMlClient();

async function importDataset() {
  // Construct request
  const request = {
    name: client.datasetPath(projectId, location, datasetId),
    inputConfig: {
      gcsSource: {
        inputUris: path.split(','),
      },
    },
  };

  // Import dataset
  console.log('Proccessing import');
  const [operation] = await client.importData(request);

  // Wait for operation to complete.
  // const [response, error] = await operation.promise();
  // console.log(`Dataset imported: ${response}`);

  // test
  operation.promise()
    .then(response => {
      console.log(`SUCCESS: Dataset imported: ${response}`);
    })
    .catch(error => {
      console.log("ERROR .. ", error);
    })
}

importDataset();